README.md

```diff
@@ -0,0 +1,97 @@
+# Temporal 
+
+This is a template for running a Temporal cluster on Render. It includes an example Go app that interacts with the cluster. Fork this repo and then click the button below to try it out!
+
+[![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy)
+
+This repo defines a [Render Blueprint](https://render.com/docs/blueprint-spec) with the following components:
+- Temporal cluster:
+  - Two Postgres databases, `temporal-db` and `temporal-db-visibility`.
+  - A `temporal-all` server that runs all Temporal services.
+  - `temporal-ui` provides the Temporal web UI.
+- Example app (based on Temporal's [Go SDK Example](https://github.com/temporalio/money-transfer-project-template-go)):
+  - `app-workflow-trigger` runs a simple HTTP server with two routes:
+    - `/` for health checking.
+    - `/trigger-workflow` for kicking off the `TransferMoney` workflow.
+  - `app-worker` executes any triggered workflows.
+
+For a more production-ready setup, use the Blueprint defined in [render.with_scaling.yaml](render.with_scaling.yaml) rather than the one in [render.yaml](render.yaml). In this Blueprint, each of the four Temporal services are run as its own Render service that can be scaled independently.It also comes with an ElasticSearch instance configured. In your fork, run
```

Comment:
```suggestion
For a more production-ready setup, use the Blueprint defined in [render.with_scaling.yaml](render.with_scaling.yaml) rather than the one in [render.yaml](render.yaml). In this Blueprint, each of the four Temporal services is run as its own Render service that can be scaled independently.It also comes with an ElasticSearch instance configured. In your fork, run
```